backend/opre_ops/ops_site/portfolios/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from opre_ops.ops_site.cans.models import CommonAccountingNumber
+
+
+class Portfolio(models.Model):
+    """
+    A Portfolio is ___
+    """
+
+    PORTFOLIO_STATUS = [
+        ("In-Process", "In-Process"),
+        ("Not-Started", "Not-Started"),
+        ("Sandbox", "Sandbox"),
+    ]
+
+    name = models.CharField(max_length=120)
+    description = models.TextField(default="", blank=True)
+    status = models.CharField(max_length=30, choices=PORTFOLIO_STATUS)
+    cans = models.ManyToManyField(CommonAccountingNumber)
+    current_fiscal_year_funding = models.DecimalField(max_digits=12, decimal_places=2)
```

Comment:
Is the idea that this will be eventually replaced with us traversing down the CAN list to get the funding?

Reply:
Yes, this was just a "placeholder" for now until we know exactly.